### ISSUE TITLE
Open recent project files doesn't work (snippets error)

### ISSUE THREAD

**Issue by zeronone** [hlissner/doom-emacs#702]
### Observed behavior

1. Open `doom-emacs`
2. Open recent project
3. Select project
4. Select file (error)

<details>
<summary>Stack Trace</summary>

```
Debugger entered--Lisp error: (void-function +snippets|enable-project-modes)
  +snippets|enable-project-modes(+data-vagrant-mode t)
  run-hook-with-args(+snippets|enable-project-modes +data-vagrant-mode t)
  (if (not +data-vagrant-mode) nil (run-hook-with-args 'doom-project-hook '+data-vagrant-mode +data-vagrant-mode) nil nil)
  (let ((last-message (current-message))) (setq +data-vagrant-mode (if (eq arg 'toggle) (not +data-vagrant-mode) (> (prefix-numeric-value arg) 0))) (if (not +data-vagrant-mode) nil (run-hook-with-args 'doom-project-hook '+data-vagrant-mode +data-vagrant-mode) nil nil) (run-hooks '+data-vagrant-mode-hook (if +data-vagrant-mode '+data-vagrant-mode-on-hook '+data-vagrant-mode-off-hook)) (if (called-interactively-p 'any) (progn nil (if (and (current-message) (not (equal last-message (current-message)))) nil (let ((local " in current buffer")) (message "+Data-Vagrant mode %sabled%s" (if +data-vagrant-mode "en" "dis") local))))))
  +data-vagrant-mode(1)
  (and (fboundp '+data-vagrant-mode) (not (and (boundp '+data-vagrant-mode) +data-vagrant-mode)) (and buffer-file-name (not (file-remote-p buffer-file-name))) t (and (file-exists-p (expand-file-name "Vagrantfile" (doom-project-root)))) t (+data-vagrant-mode 1))
  doom--init-mode-+data-vagrant-mode()
  run-hooks(after-change-major-mode-hook)
  run-mode-hooks(haskell-mode-hook)
  haskell-mode()
  set-auto-mode-0(haskell-mode nil)
  set-auto-mode()
  normal-mode(t)
  #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>)(nil t)
  apply(#f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) (nil t))
  #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert)(nil t)
  apply(#f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) (nil t))
  #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace)(nil t)
  apply(#f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) (nil t))
  #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf)(nil t)
  apply(#f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf) (nil t))
  #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf) doom|transient-hook--load-smartparens)(nil t)
  apply(#f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf) doom|transient-hook--load-smartparens) (nil t))
  #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf) doom|transient-hook--load-smartparens) doom|transient-hook--load-undo-tree)(nil t)
  apply(#f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(advice-wrapper :before #f(compiled-function (&optional error warn noauto after-find-file-from-revert-buffer nomodes) "Called after finding a file and by the default revert function.\nSets buffer mode, parses local variables.\nOptional args ERROR, WARN, and NOAUTO: ERROR non-nil means there was an\nerror in reading the file.  WARN non-nil means warn if there\nexists an auto-save file more recent than the visited file.\nNOAUTO means don't mess with auto-save mode.\nFourth arg AFTER-FIND-FILE-FROM-REVERT-BUFFER is ignored\n(see `revert-buffer-in-progress-p' for similar functionality).\nFifth arg NOMODES non-nil means don't alter the file's modes.\nFinishes by calling the functions in `find-file-hook'\nunless NOMODES is non-nil." #<bytecode 0x4009346f>) doom|transient-hook--load-autorevert) doom|transient-hook--load-saveplace) doom|transient-hook--load-recentf) doom|transient-hook--load-smartparens) doom|transient-hook--load-undo-tree) (nil t))
  after-find-file(nil t)
  find-file-noselect-1(#<buffer Functor.hs> "~/myfiles/practice/fp-course/src/Course/Functor.hs" nil nil "~/myfiles/practice/fp-course/src/Course/Functor.hs" (4305362866 16777220))
  find-file-noselect("/Users/st21371/myfiles/practice/fp-course/src/Course/Functor.hs" nil nil nil)
  find-file("/Users/st21371/myfiles/practice/fp-course/src/Course/Functor.hs")
  counsel-projectile-find-file-action("src/Course/Functor.hs")
  ivy-call()
  ivy-read("[fp-course] Find file: " (".stack-work/intero/intero-scripthauw1o" ".stack-work/intero/intero-scripteTGCEq" ".stack-work/intero/intero-scriptZ0y201" ".stack-work/intero/intero-scriptZnOPbA" ".dir-locals.el" ".stack-work/intero/intero-script8AI1aK" ".stack-work/intero/intero-scriptaeOSUY" ".stack-work/intero/intero-scriptt7RuDp" ".stack-work/intero/intero-script7kFgwL" "course.cabal" ".stack-work/intero/intero-scriptzZNaYJ" ".editorconfig" ".gitignore" ".travis.yml" "CHANGELOG.md" "CHEATSHEET.md" "Makefile" "README.md" "Setup.lhs" "Vagrantfile" "ci/ci.nix" "ci/jobsets.json" "ci/jobsets.nix" "etc/CONTRIBUTORS" "etc/LICENCE" "ops/README.md" "ops/ansible.yaml" "ops/emacs.d/init.el" "ops/haskell.yaml" "ops/sublime.yaml" "ops/vs-code.yaml" "package.yaml" "projects/NetworkServer/NetworkServer.markdown" "projects/NetworkServer/haskell/.gitignore" "projects/NetworkServer/haskell/Makefile" "projects/NetworkServer/haskell/Setup.lhs" "projects/NetworkServer/haskell/app/Chat/Main.hs" "projects/NetworkServer/haskell/app/TicTacToe/Main.hs" "projects/NetworkServer/haskell/etc/LICENCE" "projects/NetworkServer/haskell/package.yaml" "projects/NetworkServer/haskell/src/Data/TicTacToe.hs" "projects/NetworkServer/haskell/src/Network/Server.hs" "projects/NetworkServer/haskell/src/Network/Server/Chat.hs" "projects/NetworkServer/haskell/src/Network/Server/Chat/Chat.hs" "projects/NetworkServer/haskell/src/Network/Server/Chat/Loop.hs" "projects/NetworkServer/haskell/src/Network/Server/Common.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/Accept.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/Env.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/HandleLens.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/Lens.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/Line.hs" "projects/NetworkServer/haskell/src/Network/Server/Common/Ref.hs" "projects/NetworkServer/haskell/src/Network/Server/TicTacToe.hs" "projects/NetworkServer/haskell/src/Network/Server/TicTacToe/Game.hs" "projects/NetworkServer/haskell/src/Network/Server/TicTacToe/Loop.hs" "projects/NetworkServer/haskell/stack.yaml" "projects/NetworkServer/haskell/test/.gitignore" "projects/NetworkServer/haskell/test/doctests.hs" "projects/TicTacToe/TicTacToe.markdown" "share/a.txt" "share/b.txt" "share/c.txt" "share/files.txt" "src/.ghci" "src/Course.hs" "src/Course/.ghci" "src/Course/Anagrams.hs" "src/Course/Applicative.hs" "src/Course/Cheque.hs" "src/Course/Comonad.hs" "src/Course/Compose.hs" "src/Course/Core.hs" "src/Course/ExactlyOne.hs" "src/Course/Extend.hs" "src/Course/FastAnagrams.hs" "src/Course/FileIO.hs" "src/Course/Functor.hs" "src/Course/Interactive.hs" "src/Course/JsonParser.hs" "src/Course/JsonValue.hs" "src/Course/List.hs" "src/Course/ListZipper.hs" "src/Course/Monad.hs" "src/Course/MonadTutorial.hs" "src/Course/MoreParser.hs" "src/Course/Optional.hs" "src/Course/Parser.hs" "src/Course/Person.hs" "src/Course/State.hs" "src/Course/StateT.hs" "src/Course/Traversable.hs" "src/Course/Validation.hs" "stack-snapshot.yaml" "stack.yaml" "test/.gitignore" "test/Course/ApplicativeSpec.hs" "test/Course/ComonadSpec.hs" "test/Course/ExtendSpec.hs" "test/Course/FunctorSpec.hs" "test/Course/Gens.hs" "test/Course/JsonParserSpec.hs" "test/Course/ListSpec.hs" "test/Course/ListZipperSpec.hs" "test/Course/MonadSpec.hs" "test/Course/OptionalSpec.hs" "test/Course/StateSpec.hs" "test/Course/StateTSpec.hs" "test/Course/ValidationSpec.hs" "test/Spec.hs") :matcher counsel--find-file-matcher :require-match t :sort t :action (1 ("o" counsel-projectile-find-file-action "current window") ("j" counsel-projectile-find-file-action-other-window "other window") ("x" counsel-projectile-find-file-action-extern "open externally") ("r" counsel-projectile-find-file-action-root "open as root") ("m" counsel-projectile-find-file-action-find-file-manually "find file manually") ("p" (lambda (_) (counsel-projectile-switch-project)) "switch project")) :caller counsel-projectile-find-file)
  counsel-projectile-find-file(nil)
  funcall-interactively(counsel-projectile-find-file nil)
  call-interactively(counsel-projectile-find-file)
  (let (projectile-project-name projectile-require-project-root projectile-cached-buffer-file-name projectile-cached-project-root) (call-interactively (or (command-remapping (function projectile-find-file)) (function projectile-find-file))))
  (let ((default-directory dir)) (let (projectile-project-name projectile-require-project-root projectile-cached-buffer-file-name projectile-cached-project-root) (call-interactively (or (command-remapping (function projectile-find-file)) (function projectile-find-file)))))
  doom-project-find-file("~/myfiles/practice/fp-course/")
  funcall(doom-project-find-file "~/myfiles/practice/fp-course/")
  (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir))
  (if (+workspace-buffer-list) (let* (persp-p (persp (let* ((default-directory +workspaces--project-dir) (project-name (doom-project-name 'nocache))) (or (setq persp-p (+workspace-get project-name t)) (+workspace-new project-name)))) (new-name (progn (or (and (memq (type-of persp) cl-struct-perspective-tags) t) (signal 'wrong-type-argument (list 'perspective persp))) (aref persp 1)))) (+workspace-switch new-name) (if persp-p nil (switch-to-buffer (doom-fallback-buffer))) (save-current-buffer (set-buffer (doom-fallback-buffer)) (setq default-directory +workspaces--project-dir)) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir)) (+workspace-message (format "Switched to '%s' in new workspace" new-name) 'success)) (save-current-buffer (set-buffer (switch-to-buffer (doom-fallback-buffer))) (setq default-directory +workspaces--project-dir) (message "Switched to '%s'" (doom-project-name 'nocache))) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir)))
  (unwind-protect (if (+workspace-buffer-list) (let* (persp-p (persp (let* ((default-directory +workspaces--project-dir) (project-name (doom-project-name 'nocache))) (or (setq persp-p (+workspace-get project-name t)) (+workspace-new project-name)))) (new-name (progn (or (and (memq (type-of persp) cl-struct-perspective-tags) t) (signal 'wrong-type-argument (list 'perspective persp))) (aref persp 1)))) (+workspace-switch new-name) (if persp-p nil (switch-to-buffer (doom-fallback-buffer))) (save-current-buffer (set-buffer (doom-fallback-buffer)) (setq default-directory +workspaces--project-dir)) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir)) (+workspace-message (format "Switched to '%s' in new workspace" new-name) 'success)) (save-current-buffer (set-buffer (switch-to-buffer (doom-fallback-buffer))) (setq default-directory +workspaces--project-dir) (message "Switched to '%s'" (doom-project-name 'nocache))) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir))) (setq +workspaces--project-dir nil))
  (progn (unwind-protect (if (+workspace-buffer-list) (let* (persp-p (persp (let* ((default-directory +workspaces--project-dir) (project-name (doom-project-name 'nocache))) (or (setq persp-p (+workspace-get project-name t)) (+workspace-new project-name)))) (new-name (progn (or (and (memq (type-of persp) cl-struct-perspective-tags) t) (signal 'wrong-type-argument (list 'perspective persp))) (aref persp 1)))) (+workspace-switch new-name) (if persp-p nil (switch-to-buffer (doom-fallback-buffer))) (save-current-buffer (set-buffer (doom-fallback-buffer)) (setq default-directory +workspaces--project-dir)) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir)) (+workspace-message (format "Switched to '%s' in new workspace" new-name) 'success)) (save-current-buffer (set-buffer (switch-to-buffer (doom-fallback-buffer))) (setq default-directory +workspaces--project-dir) (message "Switched to '%s'" (doom-project-name 'nocache))) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir))) (setq +workspaces--project-dir nil)))
  (if (and persp-mode +workspaces--project-dir) (progn (unwind-protect (if (+workspace-buffer-list) (let* (persp-p (persp (let* ((default-directory +workspaces--project-dir) (project-name (doom-project-name 'nocache))) (or (setq persp-p (+workspace-get project-name t)) (+workspace-new project-name)))) (new-name (progn (or (and (memq (type-of persp) cl-struct-perspective-tags) t) (signal 'wrong-type-argument (list 'perspective persp))) (aref persp 1)))) (+workspace-switch new-name) (if persp-p nil (switch-to-buffer (doom-fallback-buffer))) (save-current-buffer (set-buffer (doom-fallback-buffer)) (setq default-directory +workspaces--project-dir)) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir)) (+workspace-message (format "Switched to '%s' in new workspace" new-name) 'success)) (save-current-buffer (set-buffer (switch-to-buffer (doom-fallback-buffer))) (setq default-directory +workspaces--project-dir) (message "Switched to '%s'" (doom-project-name 'nocache))) (if current-prefix-arg nil (funcall +workspaces-switch-project-function +workspaces--project-dir))) (setq +workspaces--project-dir nil))))
  +workspaces|switch-to-project("~/myfiles/practice/fp-course/")
  ivy-call()
  ivy-read("[.emacs.d] Switch to project: " ("~/myfiles/practice/fp-course/" "~/Dropbox/orgs/line/" "~/Dropbox/orgs/line-wiki/" "~/Dropbox/orgs/" "~/myfiles/practice/99problems/" "~/myfiles/dotfiles/" "~/" "~/.emacs.d/" "~/myfiles/practice/chrisdone/" "~/Dropbox/orgs/personal-wiki/" "~/Dropbox/" "/usr/local/Cellar/emacs-plus/26.1/share/emacs/26.1/lisp/" "~/work/code/talk-care-channel-web-application/" "~/Dropbox/orgs/notes/" "~/work/code/TalkCare-server-java/" "~/myfiles/stackage-everything/" "~/work/practice/" "~/work/" "~/myfiles/practice/" "~/myfiles/" "~/Downloads/spring-session-2.0.2.RELEASE/") :preselect "~/.emacs.d/" :action +workspaces|switch-to-project :require-match t :sort t :caller counsel-projectile-switch-project)
  counsel-projectile-switch-project()
  funcall-interactively(counsel-projectile-switch-project)
  call-interactively(counsel-projectile-switch-project)
  (lambda (_) (call-interactively (or (command-remapping (function projectile-switch-project)) (function projectile-switch-project))))(#<marker (moves after insertion) at 1779 in *doom*>)
  button-activate(#<marker (moves after insertion) at 1779 in *doom*> nil)
  push-button(1785)
  funcall-interactively(push-button 1785)
  call-interactively(push-button nil nil)
  command-execute(push-button)

```
</details>

### System information
<details>
<summary>Click to expand</summary>

- OS: darwin (x86_64-apple-darwin17.5.0)
- Emacs: 26.1 (May 30, 2018)
- Doom: 2.0.9 (develop 6ce30d0116c5dea4935212b35f6d76aa93aa5766)
- Graphic display: t (daemon: nil)
- System features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES THREADS
- Details:
  ```elisp
  elc count: 1
  uname -a:  Darwin LF02Q331602A90M.local 17.5.0 Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64 x86_64
  modules:   (:feature (evil +everywhere) file-templates (lookup +devdocs +docsets) snippets spellcheck (syntax-checker +childframe) version-control workspaces :completion (ivy +fuzzy) :ui doom-dashboard doom-modeline doom-quit evil-goggles fci hl-todo nav-flash neotree (popup +all +defaults) vi-tilde-fringe window-select :emacs ediff electric eshell imenu term :tools gist macos make magit pdf rotate-text tmux upload wakatime :lang (cc +irony +rtags) common-lisp clojure data erlang elixir elm emacs-lisp go (haskell +intero) javascript lua markdown nix ocaml (org +attach +babel +capture +export +present) perl purescript python rest ruby rust scala sh swift web :collab :config)
  packages:  n/a
  exec-path: (~/.cargo/bin/ /usr/local/bin/ /usr/bin/ /bin/ /usr/sbin/ /sbin/ ~/.gem/bin/ ~/.cargo/bin/ ~/.local/bin/ ~/bin/ ~/.sdkman/candidates/springboot/current/bin/ ~/.sdkman/candidates/gradle/current/bin/ ~/Library/Android/sdk/tools/ ~/Library/Android/sdk/platform-tools/ ~/n/bin/ /usr/local/Cellar/emacs-plus/26.1/libexec/emacs/26.1/x86_64-apple-darwin17.5.0/)
  ```
</details>

**Comment by hlissner** [comment]
This was fixed by #705, thanks to @AloisJanick! I forgot to autoload the hook. Let me know if it gives you any more trouble!

**Comment by zeronone** [comment]
@hlissner Thanks it fixed the problem.
